pom.xml

```diff
@@ -1,57 +1,57 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Please don't make any changes unless you are modifying/adding with your own code. 
Here it looks like you might have formatted the code.

Reply:
Ok, By Default IntelliJ IDEA format the codes.